### PR TITLE
Fix node plugin integration tests

### PR DESCRIPTION
**Pull request by wmb-software-consulting**
## Why are these changes needed?

Fix node plugin integration tests

## Checks

- [x] I've made sure the lint is passing in this PR.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- Testing Strategy
   - [ ] Unit tests
   - [x] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: inabox/deploy/deploy.go [hunk 1]
```diff
@@ -218,7 +218,10 @@ func (env *Config) RunNodePluginBinary(operation string, operator OperatorVars)
 		"NODE_BLS_OPERATOR_STATE_RETRIVER=" + operator.NODE_BLS_OPERATOR_STATE_RETRIVER,
 		"NODE_EIGENDA_SERVICE_MANAGER=" + operator.NODE_EIGENDA_SERVICE_MANAGER,
 		"NODE_CHURNER_URL=" + operator.NODE_CHURNER_URL,
-	})
+		"NODE_NUM_CONFIRMATIONS=0",
```
Comment (jianoaix) [inline inabox/deploy/deploy.go:221]: How does this affect the test?

**Review by jianoaix — COMMENTED** [review]
lgtm

File: inabox/deploy/deploy.go (hunk 1 above)
Comment (wmb-software-consulting) [reply inabox/deploy/deploy.go:221]: Right here : https://github.com/Layr-Labs/eigenda/blob/7f60e15be82c41709f4a072be016b445aba54f95/node/plugin/config.go#L96

**Review by mooselumph — APPROVED** [review]

File: inabox/deploy/deploy.go (hunk 1 above)
Comment (wmb-software-consulting) [reply inabox/deploy/deploy.go:221]: The anvil will never have 3 confirmations, so then it get stuck in a infinite loop.
https://github.com/Layr-Labs/eigenda/blob/7f60e15be82c41709f4a072be016b445aba54f95/common/geth/client.go#L236

Comment (jianoaix) [reply inabox/deploy/deploy.go:221]: I see, thanks

**Pull request merged**